app/lib/cds_importer/entity_mapper/base_mapper.rb

```diff
@@ -57,20 +76,11 @@ def before_oplog_inserts(&block)
         def before_building_model(&block)
           before_building_model_callbacks << block
         end
-      end
 
-      NATIONAL = 'N'.freeze
-      APPROVED_FLAG = '1'.freeze
-      STOPPED_FLAG = '1'.freeze
-      PATH_SEPARATOR = '.'.freeze
-      METAINFO = 'metainfo'.freeze
-      BASE_MAPPING = {
-        'validityStartDate' => :validity_start_date,
-        'validityEndDate' => :validity_end_date,
-        'metainfo.origin' => :national,
-        'metainfo.opType' => :operation,
-        'metainfo.transactionDate' => :operation_date,
-      }.freeze
+        def primary?
```

Comment:
maybe a more explicit name? primary what?

Reply:
primary mapper

Reply:
I might add a comment

Reply:
Added a comment and tidied this up so the derived_entity_class is separate from the business logic method